pkg/api/v20191231preview/openshiftcluster_validatestatic_test.go

```diff
@@ -206,22 +216,17 @@ func TestOpenShiftClusterStaticValidateProperties(t *testing.T) {
 			},
 			wantErr: "400: InvalidParameter: properties.workerProfiles: There should be exactly one worker profile.",
 		},
-	}
+	}...)
 
-	runTests(t, tests, false)
+	runTests(t, true, createTests)
+	runTests(t, false, commonTests)
```

Comment:
I wonder whether the following might be a little clearer
```
commonTests := []*validateTest{...}
createTests := []*validateTest{...}

runTests(t, enumUpdate, commonTests)
runTests(t, enumCreate, commonTests)
runTests(t, enumCreate, createTests)
```

I mean, I actually had to check the Go spec to be sure that it guarantees that your append won't have a side-effect on commonTests.